index.html

```diff
@@ -42,79 +42,361 @@ <h1>I’m Alexandra. Glad to see you!</h1>
+                  fill="#505F79"
+                /></svg
+            ></a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <!----------------- Start of Portfolio works  ----------->
+
+    <section class="works_container">
+      <div class="card_works">
+        <img
+          src="images/snapshot/Snapshoot Portfolio.png"
+          alt="Snapshoot Portfolio"
+          class="portfolio_img"
+        />
+        <h2 class="card-text_title">Tonic</h2>
+        <div class="cards frame2">
+          <img src="images/frames/Frame 2.png" alt="frame2-img" />
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages_list">
+          <li class="html">HTML</li>
+          <li class="css">CSS</li>
+          <li class="javascript">Javascript</li>
+        </ul>
+        <button type="button" class="see_button">See project</button>
+      </div>
+
+      <div class="card_works">
+        <img
+          src="images/snapshot/Snapshoot Portfolio(1).png"
+          alt="Snapshoot Portfolio"
+          class="portfolio_img"
+        />
+        <h2 class="card-text_title">Multi-Post Stories</h2>
+        <div class="cards frame2">
+          <img src="images/frames/Frame 2.png" alt="frame2-img" />
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages_list">
+          <li class="html">HTML</li>
+          <li class="css">CSS</li>
+          <li class="javascript">Javascript</li>
+        </ul>
+        <button type="button" class="see_button">See project</button>
+      </div>
+
+      <div class="card_works">
+        <img
+          src="images/snapshot/Snapshoot Portfolio(2).png"
+          alt="Snapshoot Portfolio"
+          class="portfolio_img"
+        />
+        <h2 class="card-text_title">Tonic</h2>
+        <div class="cards frame2">
+          <img src="images/frames/Frame 2.png" alt="frame2-img" />
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages_list">
+          <li class="html">HTML</li>
+          <li class="css">CSS</li>
+          <li class="javascript">Javascript</li>
+        </ul>
+        <button type="button" class="see_button">See project</button>
+      </div>
+
+      <div class="card_works">
+        <img
+          src="images/snapshot/Snapshoot Portfolio(3).png"
+          alt="Snapshoot Portfolio"
+          class="portfolio_img"
```

Comment:
- [ ] Excellent job thus far, however we have yet to concentrate on something to make your project look more professional. I noticed that your cards are not aligned well , and there is a big different if we compare with the figma design :wink:  , kindly fix this :point_down: 
Your Design 
![Screenshot from 2023-03-25 18-08-47](https://user-images.githubusercontent.com/88894525/227728757-c91a7bef-3d66-480d-a0cd-d1a0b0b78402.png)


Figma Design 
![Screenshot from 2023-03-25 18-02-22](https://user-images.githubusercontent.com/88894525/227728771-416027cf-a005-463d-8b7a-d039d7dd3753.png)

Reply:
Many thanks for the feedback. I originally thought the project out will be viewed in a mobile mood. Since the development of this project, I always right-click on the browser and go to inspect and view in mobile mode. kindly advise if this method is bad.